roles/bind_config/vars/main.yaml

```diff
@@ -0,0 +1,29 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+---
+
+# Pre-processing and validation of parameters.
+
+_bind_config_install_ferm_svc: |-
+  {{ bind_config__install_ferm_svc | d(False) | bool }}
+
+_bind_config_ferm_allow_from: |-
+  {{
+    None
+    if not bind_config__ferm_allow_from else
+    [bind_config__ferm_allow_from]
+    if bind_config__ferm_allow_from is not iterable or
+    bind_config__ferm_allow_from is string
```

Comment:
We use this in a few places (both in this repo, and roles repo), would it make sense to create a custom filter later on?

Reply:
that is a good point.. an `is_list` filter would already be very useful, I don't understand why nobody has done it yet. And a special-purpose filter to convert to list when needed would reduce duplication too

Reply:
Draft in Tina-pm/ansible-roles#38